packages/next/src/client/components/react-dev-overlay/internal/container/Errors.tsx

```diff
@@ -25,7 +25,7 @@ import { VersionStalenessInfo } from '../components/VersionStalenessInfo'
 import type { VersionInfo } from '../../../../../server/dev/parse-version-info'
 import { getErrorSource } from '../../../../../shared/lib/error-source'
 import { HotlinkedText } from '../components/hot-linked-text'
-import { PseudoHtmlDiff } from './RuntimeError/component-stack-pseudo-html'
+import { PseudoHtmlDiff } from './RuntimeError/ComponentStackPseudoHtml'
```

Comment:
it was intended to keep filename as lowercase with dashes to avoid the issues of upper/lowercase filename issues. The existing ones are pretty old that remained as camel cases. The new ones like hot-linked-text, app-router are all in lower cases with dash. We should keep this one and also the collapse-icon, and rest we can refactor later when necessary